### ISSUE TITLE
[ru_RU/PersonProvider] Double `а` in female lastName when using the `name()` function

### ISSUE THREAD

**Issue by gam6itko** [FakerPHP/Faker#832]
### Versions

|                | Version |
|:---------------|:--------|
| PHP            | 8.3   |
| `fakerphp/faker`  | v1.23.0  |

### Self-enclosed code snippet for reproduction

```php
 $generator = Factory::create('ru_RU');
 $generator->seed(1917);
 self::assertSame('Павлова Нонна Александровна', $generator->name('female')); // this fails
```

![image](https://github.com/FakerPHP/Faker/assets/3841197/25a67a0d-693c-446d-8fc6-23d51e7dd8fc)

TestCase
```php
 /**
     * @dataProvider dataIssue832
     */
    public function testIssue832(int $seed, string $expected): void
    {
        $generator = new Generator();
        $generator->seed($seed);

        foreach ($this->getProviders() as $provider) {
            $generator->addProvider($provider);
        }

        self::assertSame($expected, $generator->name('female'));
    }

    public static function dataIssue832(): iterable
    {
        //bad cases
        yield 'seed: 55' => [
            55,
            'Ларионова Алина Ивановна'
        ];
        yield 'seed: 512' => [
            512,
            'Тихонова Владлена Львовна'
        ];
        yield 'seed: 625' => [
            625,
            'Ларионова Нонна Максимовна'
        ];
        yield 'seed: 1917' => [
            1917,
            'Павлова Нонна Александровна'
        ];

        //good cases
        yield 'seed: 4' => [
            4,
            'Фаина Фёдоровна Романоваа'
        ];
        yield 'seed: 8' => [
            8,
            'Яна Александровна Жукова'
        ];
        yield 'seed: 42' => [
            42,
            'Гуляева Розалина Дмитриевна'
        ];
    }
```

**Comment by pimjansen** [comment]
There is only an assumption that a seed will return the same value on the same version of the code on the same server etc.

If you call the name twice while seeding, are you receiving the same outcome?

**Comment by gam6itko** [comment]
> If you call the name twice while seeding, are you receiving the same outcome?

No, subsequent calls of methods will return a different result.

**Comment by gam6itko** [comment]
Fixed in #836
